libs/application/template-api-modules/src/lib/modules/templates/children-residence-change/children-residence-change.service.ts

```diff
@@ -104,13 +104,14 @@ export class ChildrenResidenceChangeService {
 
     participants.push(parentA, parentB)
 
+    const durationType = answers.durationType as string
```

Comment:
I don't like this but it's needed because `durationType` is set as optional in the dataschema..

Reply:
Agree, but don't see a better solution currently 😞 
if we would mix this logic in on line 112 then the code would become pretty messy I think.